store/etcd/proto.go

```diff
@@ -168,16 +163,24 @@ func valueFromProtobuf(v *pb.Value) *rule.Value {
 	}
 }
 
+// signatureToProtobuf creates a protobuf Signature from a regula.Signature.
+func signatureToProtobuf(sig *regula.Signature) *pb.Signature {
+	return &pb.Signature{
+		ReturnType: sig.ReturnType,
+		ParamTypes: sig.ParamTypes,
```

Comment:
If the signature is updated (especially the `ParamTypes` map) after being passed to this function, the `pb.Signature` will also be updated. Do we really want this?

Reply:
It makes sense, it's just that for now, it's useless to copy it just in case.